### PR TITLE
Add a simple CNN

### PR DESCRIPTION
**Pull request by kongsgard**
Graph:
Convolutional Layer #1
Convolutional Layer #2
Dense layer
Dropout
Readout layer

### CODE REVIEW

File: configs/simple_CNN.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "exp_name": "simple_CNN",
+  "num_epochs": 10,
+  "num_iter_per_epoch": 100,
```
Comment (kongsgard) [inline configs/simple_CNN.json:4]: Number of iterations equal 50 gives similar results, but double the performance. Please change.
